src/shared/types/api/tokenValidator.ts

```
@@ -0,0 +1,37 @@
+import { axiosClient } from './http-client';
+import useAuthStore from '../../../app/provider/authStore';
+
+// 토큰 유효성 검증 함수 (httpOnly 쿠키 사용)
+export const validateToken = async (): Promise<boolean> => {
+  try {
+    // 토큰 유효성 검증을 위한 API 호출 (사용자 정보 조회)
+    await axiosClient.get('/users');
+    return true;
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)